sphinx/themes/basic/layout.html

```diff
@@ -153,19 +153,22 @@ <h3>{{ _('Navigation') }}</h3>
 {%- block extrahead %} {% endblock %}
   </head>
   {%- block body_tag %}<body>{% endblock %}
+
+{%- block skip_to_main_content %}<a id="sphinx-skip-link" href="#sphinx-skip-target">Skip to main content</a>{% endblock %}
+
 {%- block header %}{% endblock %}
 
 {%- block relbar1 %}{{ relbar() }}{% endblock %}
 
 {%- block content %}
   {%- block sidebar1 %} {# possible location for sidebar #} {% endblock %}
-
     <div class="document">
   {%- block document %}
       <div class="documentwrapper">
       {%- if render_sidebar %}
         <div class="bodywrapper">
       {%- endif %}
+          <span id="sphinx-skip-target"></span>
```

Comment:
I am a bit confused why a `<span id="sphinx-skip-target"></span>` is being added instead of an `id` (like `main-content` or something similar) to `<div class="body" role="main">`?

The skip link should take the user to the beginning of the `main` content area, here it would take them to a `span` just _before_ this which is not the same.

Reply:
Fixed